setup.sh

```diff
@@ -0,0 +1,295 @@
+    OS="Debian"
+    PM="apt-get"
+    MailScannerDownloadPath="https://s3.amazonaws.com/msv5/release/MailScanner-$MailScannerVersion.deb.tar.gz"
+else
+    echo "OS NOT SUPPORTED - Please perform a manual install"
+    exit 1;
+fi
+
+EndNotice=""
+
+
+function logprint {
+    echo "$1"
+    echo "$1" >> /root/mailwatchInstall.log
+}
+logprint "Clearing temp dir"
+rm -rf /tmp/mailwatchinstall/*
+
+if ! ( type "wget" > /dev/null 2>&1 ) ; then
+    $PM install wget
+fi
+
+read -p "Install/upgrade MailScanner version $MailScannerVersion?:(y/n)[y]: " installMailScanner
+if [ -z $installMailScanner ] || [ "$installMailScanner" == "y" ]; then
+    logprint "Starting MailScanner install"
+    mkdir -p /tmp/mailwatchinstall/mailscanner
+    logprint "Downloading current MailScanner release $MailScannerVersion:"
+    wget -O /tmp/mailwatchinstall/mailscanner/MailScanner.deb.tar.gz  "$MailScannerDownloadPath"
+    logprint "Extracting mailscanner files:"
+    tar -xzf /tmp/mailwatchinstall/mailscanner/MailScanner.deb.tar.gz -C /tmp/mailwatchinstall/mailscanner/
+    logprint "Starting MailScanner install script"
+    /tmp/mailwatchinstall/mailscanner/MailScanner-$MailScannerVersion/install.sh
+    logprint "MailScanner install finished."
+    EndNotice="$EndNotice \n * Adjust /etc/MailScanner.conf to your needs \n * Set run_mailscanner=1 in /etc/MailScanner/defaults"
+    sleep 1
+else
+   logprint "Not installing MailScanner"
+fi
+
+logprint "Installing Encoding::FixLatin"
+cpan -i Encoding::FixLatin
+
+##ask directory for web files
+read -p "In what location should MailWatch be installed?[/var/www/mailscanner/]:" WebFolder
+if [ -z $WebFolder ]; then
+    WebFolder="/var/www/mailscanner/"
+fi
+logprint "Using web directory $WebFolder"
+
+/etc/init.d/mailscanner stop
+
+if [ -d $WebFolder ]; then
+   read -p "Folder $WebFolder already exists. Content will get deleted. Do you want to continue?(y/n)[n]: " response
+   if [ -z $response ]; then
+       logprint "Stopping setup on user request"
+       exit
+   elif [ "$response" == "n" ]; then
+       logprint "Stopping setup on user request"
+       exit
+   fi
+   logprint "Clearing web directory"
+   rm -R $WebFolder
+fi
+
+logprint "Setting up sql credentials"
+#get sql credentials
+read -p "SQL user for mailwatch[mailwatch]:" SqlUser
+if [ -z $SqlUser ]; then
+    SqlUser="mailwatch"
+fi
+read -p "SQL password for mailwatch[mailwatch]:" SqlPwd
+if [ -z $SqlPwd ]; then
+    SqlPwd="mailwatch"
+fi
+read -p "SQL database for mailwatch[mailscanner]:" SqlDb
+if [ -z $SqlDb ]; then
+    SqlDb="mailscanner"
+fi
+read -p "SQL host of database[localhost]:" SqlHost
+if [ -z $SqlHost ]; then
+    SqlHost="localhost"
+fi
+logprint "Using sql credentials user: $SqlUser; password: $SqlPwd; db: $SqlDb; host: $SqlHost"
+
+if ! ( type "mysqld" > /dev/null 2>&1 ) ; then
+    read -p "No mysql server found. Do you want to install mariadb as sql server?(y/n)[y]: " response
```

Comment:
It should be in default packages at least for debian (>=8.0) and ubuntu (>=14.04). And I think also in RH 7